selfhost/main.jakt

```diff
@@ -209,6 +210,7 @@ fn main(args: [String]) -> c_int {
     let completions = args_parser.option(["-m", "--completions"])
     let print_symbols = args_parser.flag(["--print-symbols"])
     let project_name = args_parser.option(["--create"])
+    let use_ccache = args_parser.flag(["--use-ccache"])
```

Comment:
Should the `ccache' flag be a seperate commit?

Reply:
Sure.